css/gloss.css

```diff
@@ -549,4 +549,9 @@ textarea:disabled, select:disabled {
 
 input.required, textarea.required, select.required{
     border: 1px solid black;
+}
+
+.loadingMsg{
```

Comment:
This is in `#loading` already, and this rule is where the conflicting background is set. There are `var`s as well, but this is enough for right here.